test/core/__snapshots__/snapshot.spec.js.snap

```diff
@@ -32,15 +34,16 @@ Object {
 + <span>
     <span />
 - </div>
-+ </span>",
++ </span>
+  ",
   "expected": "<div>
   <span />
 </div>",
   "pass": false,
 }
 `;
 
-exports[`test passes if new and old are null 1`] = `
+exports[`passes if new and old are null 1`] = `
```

Comment:
This test would fail if you don't remove the new properties (see my previous comment) but you can update it with `-u`.